### ISSUE TITLE
Package still referred to as 'rnetmatch' in places

### ISSUE THREAD

**Issue by Robinlovelace** [JosiahParry/anime#5]
I'm up for giving this a go. I guessed that `anime` would not be available but seems that happily it is:

```
available::available("anime")
── anime ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Name valid: ✔
Available on CRAN: ✔ 
Available on Bioconductor: ✔
Available on GitHub:  ✔ 
Abbreviations: http://www.abbreviations.com/anime
Wikipedia: https://en.wikipedia.org/wiki/anime
Wiktionary: https://en.wiktionary.org/wiki/anime
Sentiment:???
```

Will update accordingly. Big thumbs up to the new name and looking forward to using it.

**Comment by JosiahParry** [comment]
Fwiw this package cannot go on CRAN until they update their distributions of Rust. This is a limitation set by geo-rust.

**Comment by Robinlovelace** [comment]
Thanks for info. No big issue as far as I'm concerned at this point, does not affect our use case. I recall chiming into a discussion on this, hopefully CRAN will update their distribution of Rust soon, but no blocker to progressing this if not. On the case!

**Comment by Robinlovelace** [comment]
`devtools::check()` still fails, will try to get it working.

**Comment by Robinlovelace** [comment]
There are no longer any errors, just warnings:

```r
devtools::check() # inside r folder
```

```
...
✔  checking DESCRIPTION meta-information ...
W  checking top-level files ...
   A complete check needs the 'checkbashisms' script.
   See section ‘Configure and cleanup’ in the ‘Writing R Extensions’
   manual.
   Non-standard files/directories found at top level:
     ‘data-raw’ ‘dev’
✔  checking for left-over files ...
✔  checking index information
✔  checking package subdirectories ...
✔  checking code files for non-ASCII characters ...
✔  checking R files for syntax errors ...
✔  checking whether the package can be loaded ...
✔  checking whether the package can be loaded with stated dependencies ...
✔  checking whether the package can be unloaded cleanly ...
✔  checking whether the namespace can be loaded with stated dependencies ...
✔  checking whether the namespace can be unloaded cleanly ...
✔  checking loading without being on the library search path ...
W  checking dependencies in R code ...
   '::' or ':::' import not declared from: ‘wk’
   Namespaces in Imports field not imported from:
     ‘arrow’ ‘nanoarrow’
     All declared Imports should be used.
✔  checking S3 generic/method consistency ...
✔  checking replacement functions ...
✔  checking foreign function calls ...
✔  checking R code for possible problems (1.3s)
W  checking for missing documentation entries ...
   Undocumented code objects:
     ‘anime’
   All user-level objects in a package should have documentation entries.
   See chapter ‘Writing R documentation files’ in the ‘Writing R
   Extensions’ manual.
✔  checking line endings in shell scripts
✔  checking line endings in C/C++/Fortran sources/headers ...
✔  checking line endings in Makefiles ...
✔  checking compilation flags in Makevars ...
✔  checking for GNU extensions in Makefiles ...
✔  checking for portable use of $(BLAS_LIBS) and $(LAPACK_LIBS) ...
✔  checking use of PKG_*FLAGS in Makefiles ...
✔  checking use of SHLIB_OPENMP_*FLAGS in Makefiles ...
✔  checking pragmas in C/C++ headers and code ...
✔  checking compilation flags used ...
✔  checking compiled code ...
─  checking examples ... NONE
✔  checking for non-standard things in the check directory
✔  checking for detritus in the temp directory
   
   See
     ‘/tmp/RtmpuFpOQf/file5afd69462f27/anime.Rcheck/00check.log’
   for details.
   
── R CMD check results ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── anime 0.0.0.9000 ────
Duration: 1m 45.2s

❯ checking top-level files ... WARNING
  A complete check needs the 'checkbashisms' script.
  See section ‘Configure and cleanup’ in the ‘Writing R Extensions’
  manual.
  Non-standard files/directories found at top level:
    ‘data-raw’ ‘dev’

❯ checking dependencies in R code ... WARNING
  '::' or ':::' import not declared from: ‘wk’
  Namespaces in Imports field not imported from:
    ‘arrow’ ‘nanoarrow’
    All declared Imports should be used.

❯ checking for missing documentation entries ... WARNING
  Undocumented code objects:
    ‘anime’
  All user-level objects in a package should have documentation entries.
  See chapter ‘Writing R documentation files’ in the ‘Writing R
  Extensions’ manual.

❯ checking installed package size ... NOTE
    installed size is 23.6Mb
    sub-directories of 1Mb or more:
      libs  23.5Mb
```

**Comment by Robinlovelace** [comment]
No sure why it imports nanoarrow, plan to move to Suggests to make the checks happy.

**Comment by Robinlovelace** [comment]
Fixed in #6
